src/main/configurations/Translate/Configuration_AddRolToZgw.xml

```diff
@@ -71,7 +76,7 @@
                 <Param name="code" value="TechnicalError" /> <!-- codes: TechnicalError, TranslationError, ConfigurationError-->
                 <Param name="reason"
                     pattern="No roltype with matching omschrijving is found" />
-                <Param name="details" pattern="omschrijving:{TypeRolOmschrijving} cannot be found in the list of roltypes"/>
+                <Param name="details" pattern="omschrijving: {TypeRolOmschrijving} cannot be found in the list of roltypes"/>
```

Comment:
It is much easier to simply define TypeRolOmschrijving in the XsltPipe that is building the error itself as a param. A pattern tries to resolve the variable from a sessionKey or a param. The only thing to keep in mind with resolving from a param, is that it needs to be defined above the param with the pattern.

Also with ZaakBrug, always add ignoreUnresolvablePatternElements="true" when using patterns.

Example:
<Param name="TypeRolOmschrijving" xpathExpression="role/typeRolOmschrijving"/>
<Param name="details" pattern="omschrijving:{TypeRolOmschrijving} cannot be found in the list of roltypes" ignoreUnresolvablePatternElements="true" />